provisioning/src/handlers/portfolioDrafts/portfolio/post.ts

```diff
@@ -0,0 +1,42 @@
+import { Context, HttpRequest } from '@azure/functions'
+import dayjs from 'dayjs'
+import { PortfolioStep } from '../../../lib/models/PortfolioStep'
+import { PortfolioFull } from '../../../lib/models/PortfolioFull'
+import { CosmosClient } from '@azure/cosmos'
+
+export async function handler (context: Context, req: HttpRequest): Promise<void> {
+  const connectionString = process.env['COSMOS-CONNECTION-STRING']
+  context.log('Connecting to ' + connectionString)
+  const client = new CosmosClient(connectionString)
+  const id: string = req.params.portfolioDraftId
```

Comment:
Glad to see straightforward support for named path parameters here.

Reply:
Explicitly defining it the serverless.yml was giving me problems, its nice that Azure assumes the path param automagically though.